core/vm/eips.go

```diff
@@ -303,3 +304,20 @@ func enable6780(jt *JumpTable) {
 		maxStack:    maxStack(1, 0),
 	}
 }
+
+func enable4762(jt *JumpTable) {
+	jt[SSTORE].constantGas = 0
+	jt[SSTORE].dynamicGas = gasSStore4762
+	jt[SLOAD].constantGas = 0
+	jt[SLOAD].dynamicGas = gasSLoad4762
```

Comment:
sets values to 0 so that `params.WarmCost` isn't charged there. It's charged in `dynamicGas`.